homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,13 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    if(count < 1){
+        return {};
+    }
+    std::vector<int> vec;
+    vec.reserve(count);
+    for(int i=0;i<count;i++){
+        vec.push_back(step * (i+1));
+    }
+    return vec;
```

Comment:
Add space after `if`, `for`, `while` `function`  etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..